airbnb_project/airbnb_project/settings.py

```diff
@@ -82,6 +82,10 @@
     'default': {
         'NAME': 'airbnb_db',
         'ENGINE': 'django.db.backends.postgresql',
+        'USER': os.environ['POSTGRES_USER'],
```

Comment:
`nit`: It'd be nice to have an env file where we can find out what all environment variables are needed. Not blocking on it though.

Reply:
@VimanyuAgg thanks for your feedback, could it be an env template file like:
```
SECRET_KEY=
DJANGO_DEBUG=
AIRBNB_PUBLIC_API_KEY=
POSTGRES_USER=
POSTGRES_PASSWORD=
POSTGRES_DB=
POSTGRES_HOST_PORT=
```

Reply:
i will still merge this pr to let Mangat use Dockerfile